src/Workspaces/Core/Portable/Simplification/SimplificationHelpers.cs

```diff
@@ -101,15 +101,16 @@ internal static bool IsValidSymbolInfo(ISymbol symbol)
             return symbol != null && !symbol.IsErrorType();
         }
 
-        internal static bool ShouldSimplifyMemberAccessExpression(SemanticModel semanticModel, SyntaxNode expression, OptionSet optionSet)
+        internal static bool ShouldSimplifyMemberAccessExpression(
+            SemanticModel semanticModel, OptionSet optionSet, ISymbol symbol)
         {
-            var symbol = GetOriginalSymbolInfo(semanticModel, expression);
-            if (symbol == null ||
-                (!symbol.IsStatic &&
-                 (symbol.IsKind(SymbolKind.Field) && optionSet.GetOption(CodeStyleOptions.QualifyFieldAccess, semanticModel.Language).Value ||
-                 (symbol.IsKind(SymbolKind.Property) && optionSet.GetOption(CodeStyleOptions.QualifyPropertyAccess, semanticModel.Language).Value) ||
-                 (symbol.IsKind(SymbolKind.Method) && optionSet.GetOption(CodeStyleOptions.QualifyMethodAccess, semanticModel.Language).Value) ||
-                 (symbol.IsKind(SymbolKind.Event) && optionSet.GetOption(CodeStyleOptions.QualifyEventAccess, semanticModel.Language).Value))))
+            if (symbol.IsStatic)
+                return false;
```

Comment:
❔  In the old code, this case returned `true`. Was the change intentional?

Reply:
Fixed.  Thanks for catching this.  Is also revealed a test hold.  I've reverted to the proper logic and added a test to ensure that we don't regrss this desirable behavior.